packages/test/src/browser/view/test-result-widget.ts

```diff
@@ -83,6 +91,35 @@ export class TestResultWidget extends BaseWidget {
             } else {
                 this.content.append(this.node.ownerDocument.createTextNode(message.message));
             }
+            if (message.stackTrace) {
+                message.stackTrace.map(frame => this.renderFrame(frame));
+            }
+        });
+    }
+
+    renderFrame(stackFrame: TestMessageStackFrame): void {
+        const frameElement = this.node.ownerDocument.createElement('div');
+        frameElement.append(this.node.ownerDocument.createTextNode(stackFrame.label));
+
+        // Add URI information as clickable links
+        if (stackFrame.uri) {
+
+            const uri = stackFrame.uri;
+            frameElement.append(' from ');
+            const link = this.node.ownerDocument.createElement('a');
+            link.textContent = `${this.labelProvider.getName(uri)}`;
+            link.title = `${uri}`;
```

Comment:
Using `title` here instead of `href` prevents the UI from rendering the anchor tag as a proper link. No underlining and the cursor does not change to a pointing hand when hovering. Any particular reason for doing it this way?

Reply:
No, no particular reason. That's fixed.